app/dashboard/utils.py

```diff
@@ -998,3 +994,38 @@ def list_urls(lis, acc=None):
 
 def get_custom_avatars(profile):
     return CustomAvatar.objects.filter(profile=profile).order_by('-id')
+
+
+
+
+
+def getTransaction__(txid):
+    """Returns a transaction"""
+    try:
+        return web3.eth.getTransaction(txid)
+    except TransactionNotFound:
+        return None
+
+
+def getBlock__(latest):
+    try:
+        return web3.eth.getBlock(latest)
+    except BlockNotFound:
+        return None
+
+def getTransactionCount__(address):
```

Comment:
Why do we want to wrap these web3 calls in try/except blocks? Wouldn't it be better to fail loudly? https://realpython.com/the-most-diabolical-python-antipattern/

Reply:
@danlipert do you prefer to have an error treatment in each method call?